### PR TITLE
Feature/readme update

### PR DESCRIPTION
**Pull request by debora-hcalves**
The requested changes in the README have been made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,222 @@
...
+- [Working with dbt](#working-with-dbt)
+    - [Running Models](#running-models)
+    - [Running Tests](#running-tests)
+- [Project Structure](#project-structure)
+    - [Development Folders](#development-folders)
+
+
+## Data Transformation 
+
+dbt is composed of different moving parts working harmoniously. All of them are important to what dbt does — transforming data.  When you execute dbt run, you are running a model that will transform your data without that data ever leaving your warehouse.
```
Comment (sydneynotthecity) [inline README.md:21]: nit, can we put dbt run in a code snippet? `dbt run`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,222 @@
 # stellar-dbt-public
 Public DBT instance to aid in data transformation for analytics purposes
+
```
Comment (sydneynotthecity) [inline README.md:3]: Can you add a sentence that links to the dbt docs for those interested in setting up their own project?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,222 @@
...
+## Data Transformation 
+
+dbt is composed of different moving parts working harmoniously. All of them are important to what dbt does — transforming data.  When you execute dbt run, you are running a model that will transform your data without that data ever leaving your warehouse.
+
+### Workflow
+
+The top level of a dbt workflow is the project. A project is a directory of a `.yml` file (the project configuration) and either `.sql` or `.py` files (the models). The project file tells dbt the project context, and the models let dbt know how to build a specific data set.
+
+A model is a single file containing a final select statement, and a project can have multiple models, and models can even reference each other.
```
Comment (sydneynotthecity) [inline README.md:27]: Might be worthwhile to mention that the end result of a model is a data mart table, with the intent of simplifying analytics.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,222 @@
...
+  What is done in the intermediate:
+
+  - Join between different source queries
+  - More complex functions that would not enter the staging layer
+  - Aggregations
+  - Creation of business metrics/rules
+
+  What not to do in intermediate:
+
+  - Opening of sources
```
Comment (sydneynotthecity) [inline README.md:67]: Opening of staging sounds clunky. Can we reword to something like "ingesting raw source data"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,222 @@
...
+
+The models are divided into:
+
+  - Dimension tables (dim_table): where the dimensions of the models will be defined and gather all the sources of the respective dimension;
+  - Fact tables (fct_table): where the final models of the business strategy to be analyzed will be located;
+  - Aggregate tables (agg_table): where are aggregations of fact tables by one dimension.
+
+### Tests
+
+Tests are assertions you make about your models and other resources in your dbt project (e.g. sources, seeds and snapshots). When you run dbt test, dbt will tell you if each test in your project passes or fails. Like almost everything in dbt, tests are SQL queries. In particular, they are select statements that seek to grab "failing" records, ones that disprove your assertion.
```
Comment (sydneynotthecity) [inline README.md:94]: Put `dbt test` in a code snippet

File: README.md [hunk 6]
```diff
@@ -1,2 +1,222 @@
...
+    - [Oauth connection](#oauth)
+- [dbt Setup](#dbt-setup)    
+- [Working with dbt](#working-with-dbt)
+    - [Running Models](#running-models)
+    - [Running Tests](#running-tests)
+- [Project Structure](#project-structure)
+    - [Development Folders](#development-folders)
+
+
+## Data Transformation
```
Comment (sydneynotthecity) [inline README.md:19]: This might be better named as `dbt Overview` but I don't feel super strongly about it

**Review by sydneynotthecity — APPROVED** [review]
Looks good. Left some small feedback but it's not blocking merging the PR

**Comment by debora-hcalves** [comment]
Syd, I've made the small changes you requested. Can you please review to see if everything is okay?

**Review by sydneynotthecity — APPROVED** [review]
Looks good!

**Pull request merged**